highfive/newpr.py

```diff
@@ -111,6 +116,8 @@ def set_assignee(self, assignee, owner, repo, issue, user, author, to_mention):
         except urllib2.HTTPError, e:
             if e.code == 201:
                 pass
+            elif e.code == 400:
```

Comment:
Based on #190, these should all probably be handling for the 404 case, not a 400 case.